css/styles.css

```diff
@@ -143,8 +145,102 @@ aside p {
   padding: 0 40px 0 20px;
 }
 
+.imageBox {
+  padding: 0px 0px 0px 0px;
+}
+
+img {
+  margin-top: 60px;
+}
+
 footer {
   clear: both;
   text-align: center;
   padding: 10px;
 }
+
+#posts {
+  width: 700;
+  display: inline-block;
+  margin-right: 20px;
+}
+
+.post {
+  border-bottom: 1px solid #f2e2cd;
+  padding-bottom: 40px;
+  margin-bottom: 40px;
+}
+
+.readmore, .readless {
+  text-align: right;
+  margin-top: 30px;
+}
+.readmore a, .readless a {
+  color: #f2e2cd;
+  font-weight: 600;
+}
+
+.hide {
+  display: none;
+}
+
+@media (max-width: 900px) {
+
+aside {
```

Comment:
Indent these rules to make it more obvious they're contained within your media query.